src/appMain/hmi_capabilities.json

```diff
@@ -349,7 +349,7 @@
 },
     "TTS":
 	{
-		"capabilities":"TEXT",
+		"capabilities":["TEXT"],
```

Comment:
@dtrunov according to implementation this parameter is expected to be a single string, so I have rolled back this change came trough automatic got merge, see f9d3e34